src/invidious/jobs/bypass_captcha_job.cr

```diff
@@ -94,7 +97,8 @@ class Invidious::Jobs::BypassCaptchaJob < Invidious::Jobs::BaseJob
             loop do
               sleep 10.seconds
 
-              response = JSON.parse(captcha_client.post("/getTaskResult", body: {
+              response = JSON.parse(captcha_client.post("/getTaskResult",
+                headers: HTTP::Headers{"Content-Type" => "application/json"}, body: {
```

Comment:
Looks good, though this needs probably testing. I dont know if anti-captcha always returns correct content type

Reply:
anti-captcha doesn't care about the content type but the library that I use for parsing the body do, it's [koa-body](https://github.com/dlau/koa-body). My library doesn't want to parse the body to json unless the correct content-type is provided.

In the end, setting `Content-Type` to `application/json` is a good practice when submitting a JSON body. That's for instance what GitHub requires to do when sending requests to their rest api: https://developer.github.com/v3/media/